dipy/segment/tests/test_clustering.py

```diff
@@ -275,15 +292,27 @@ def test_cluster_map_add_cluster():
 
         clusters.add_cluster(cluster)
         assert_equal(type(cluster), Cluster)
-        assert_equal(len(clusters), i+1)
+        assert_equal(len(clusters), i + 1)
         assert_equal(cluster, clusters[-1])
 
-    assert_array_equal(list(itertools.chain(*clusters)), list(itertools.chain(*list_of_indices)))
+    assert_array_equal(
```

Comment:
I think the following would be easier to read: 

    assert_array_equal(list(itertools.chain(*clusters)), 
                                    list(itertools.chain(*list_of_indices)))